documentation/modules/proc-configuring-kafka-bridge-smr-metrics.adoc

```diff
@@ -0,0 +1,42 @@
+[id='proc-configuring-kafka-bridge-smr-metrics-{context}']
+= Configuring Strimzi Metrics Reporter metrics
+
+[role="_abstract"]
+Enable metrics with Strimzi Metrics Reporter by setting the `bridge.metrics` option to `strimziMetricsReporter`.
+
+.Prerequisites
+
+* xref:proc-downloading-kafka-bridge-{context}[The Kafka Bridge installation archive is downloaded].
+
+.Procedure
+
+. Set the `bridge.metrics` configuration to `strimziMetricsReporter`.
++
+.Configuration for enabling metrics
+
+[source,properties]
+----
+bridge.metrics=strimziMetricsReporter
+----
++
+Optionally, you can set a comma-separated list of regexes used to filter exposed metrics using the `kafka.prometheus.metrics.reporter.allowlist` property.
+If not set, all available metrics will be exposed.
+
+When needed, it is possible to configure the allowlist per client type.
+For example, set `kafka.admin.prometheus.metrics.reporter.allowlist=` to exclude all admin client metrics.
+
+In case the same property is set with multiple prefixes, the most specific always wins.
+For example, `kafka.producer.prometheus.metrics.reporter.allowlist` wins over `kafka.prometheus.metrics.reporter.allowlist`. 
+
+You can add any plugin configuration to the Bridge properties file using `kafka.`, `kafka.admin.`, `kafka.producer.`, and `kafka.consumer.` prefixes.
+See the https://github.com/strimzi/metrics-reporter[Strimzi Metrics Reporter] documentation for more details.
```

Comment:
What exactly does this mean, given we have only one set of metrics on the output?

Reply:
The Strimzi metrics reporter plugin configuration is a per-client configuration.
The bridge has 3 clients, admin, producer and consumer and you can apply a configuration to all of them by using the `kafka.` prefix or a specific config for each client using the corresponding prefix (i.e. if you want a specific config on the consumer only, you set it with the `kafka.consumer.` prefix).
For example in the above description, you could have an allowList different for admin, producer and consumer by setting it with the appropriate prefix, for example:

````
kafka.consumer.prometheus.metrics.reporter.allowlist=<some list>"
kafka.producer.prometheus.metrics.reporter.allowlist=<some other list>"
```

Reply:
For example, I could only be interested in admin client metrics for troubleshooting purpose, so I would set producer and consumer to empty allow list.

Reply:
Hmm, maybe this would deserve some clarification in the docs. Especially the `the most specific always wins` could be confusing as you can also take it to the list of the different expressions etc.